lib/ask/runtime/broker.ex

```diff
@@ -303,15 +316,20 @@ defmodule Ask.Runtime.Broker do
   end
 
   defp sync_step_internal(session, reply, session_mode) do
-    respondent = session.respondent
-
     transaction_result = Repo.transaction(fn ->
       try do
+        session = if reply == Flow.Message.answer() do
+          update_respondent_disposition(session, "contacted")
+        else
+          update_respondent_disposition(session, "started")
```

Comment:
If a respondent jumps from 'queued' to 'started', do the 'contacted' disposition history be created anyway?